carbonserver/carbonserver.go

```diff
@@ -699,52 +750,13 @@ func (listener *CarbonserverListener) expandGlobs(query string) ([]string, []boo
 	}
 	globs = append(globs, query)
 	// TODO(dgryski): move this loop into its own function + add tests
```

Comment:
will this comment be relevant after merge?

Reply:
Nice catch. :P Will see what I could do.

Reply:
Updated the todo. ;)